### PR TITLE
Add Black Friday 2024 card to the My Home screen

### PR DESCRIPTION
**Pull request by claudiucelfilip**
Related to #

## Proposed Changes

This adds a task cards for the Black Friday 2024 promo.
* the card  will be visible in the primary slider for the Post Launch view

More info: pcNC1U-1u6-p2#comment-1772

## Testing Instructions

* Sandbox `public-api.wordpress.com`
* Apply D166880-code and follow instructions
* In `client/my-sites/customer-home/cards/tasks/black-friday-2024/index.jsx` append the coupon code from `36568-pb` to the `actionUrl`

* Go to `/home/{SITE_ID}?view=VIEW_POST_LAUNCH`
* You should see the Black Friday 2024 slide in the primary slider 

Screenshot 2024-11-26 at 08 41 51

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/my-sites/customer-home/cards/tasks/black-friday-2024/index.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import blackFriday2024Illustration from 'calypso/assets/images/customer-home/illustration--black-friday-2024.jpg';
+import { TASK_BLACK_FRIDAY_2024 } from 'calypso/my-sites/customer-home/cards/constants';
+import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+
+import './style.scss';
+
+const Affiliates = () => {
```
Comment (southp) [inline client/my-sites/customer-home/cards/tasks/black-friday-2024/index.jsx]: This looks like a typo. Maybe somethjing like `BlackFriday2024`?

**Review by southp — APPROVED** [review]
LGTM. Once the component name typo is fixed, this will be good to go on the launch day with the actual coupon code added :) 

Desktop:
CleanShot 2024-11-25 at 18 49 37@2x

Mobile:
CleanShot 2024-11-25 at 18 49 51@2x

Tablet:
CleanShot 2024-11-25 at 18 51 36@2x

File: client/my-sites/customer-home/cards/tasks/black-friday-2024/index.jsx (hunk 1 above)
Comment (claudiucelfilip) [reply client/my-sites/customer-home/cards/tasks/black-friday-2024/index.jsx]: Thanks for catching that!
Fixed.

Comment (jrdnhllr) [reply client/my-sites/customer-home/cards/tasks/black-friday-2024/index.jsx]: Can we pls update the headline to "Our Biggest Sale..."

And also use this promo visual for the image: nNq0GqvlzdQ5i3OgusUon9-fi-249_65735 

Thanks!

Comment (claudiucelfilip) [reply client/my-sites/customer-home/cards/tasks/black-friday-2024/index.jsx]: @jrdnhllr Headline and promo visual have been updated.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/17020359

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @claudiucelfilip for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/17020359).

**Pull request merged**